wander-connect-sdk/src/wander-connect.ts

```diff
@@ -544,6 +551,51 @@ export class WanderConnect {
     this._close();
   }
 
+  public signOut(): Promise<void> {
+    // TODO: Check if already authenticated.
+
+    this.close();
+
+    return new Promise((resolve, reject) => {
+      const { iframeRef } = this;
+      const contentWindow = iframeRef?.contentWindow;
+
+      if (!iframeRef || !contentWindow) {
+        reject(new Error("Missing Wander Connect iframe"));
+        return;
+      }
+
+      isomorphicSendMessage({
+        destination: "background",
+        // @ts-ignore
```

Comment:
`shim.d.ts` doesn't work inside `wander-connect-sdk`, so types for `isomorphicSendMessage()` are missing.